docs/content/Caching/Getting-Started-Pre-Aggregations.md

```diff
@@ -159,7 +159,175 @@ the pre-aggregations are rebuilt.
+| 2      | 49             | 2            | 6            | 248       | 0.1               | 2021-01-20 00:00:00.000000 |
+| 3      | 89             | 3            | 6            | 197       | 0.2               | 2021-11-25 00:00:00.000000 |
+| 4      | 71             | 4            | 8            | 223       | 0.15              | 2019-12-23 00:00:00.000000 |
+| 5      | 64             | 5            | 5            | 75        | 0.7               | 2019-04-20 00:00:00.000000 |
+
+Looking at the raw data, we can see that if the data were to be aggregated by
+day (for example), then we could simply add together the `quantity` and `price`
+fields. This means that `quantity` and `price` are both "additive measures", and
+we can represent them in the `LineItems` schema as follows:
+
+```javascript
+cube(`LineItems`, {
+  ...,
+  measures: {
+    ...,
+    quantity: {
+      sql: `quantity`,
+      type: `sum`,
+    },
+    price: {
+      type: `sum`,
+      sql: `price`,
+      format: `currency`,
+    },
+  },
+  ...,
+});
+```
+
+Because neither `quantity` and `price` reference any other measures in our
+`LineItems` cube, we can also say that they are "additive leaf measures". Any
+query requesting only these two measures can be called a "leaf measure additive"
+query.
+
+### Non-Additivity
+
+Using the same sample data for `line_items`, there's a `profit_margin_pct` field
+which is different for each row. However, despite the value being numerical, it
+doesn't actually make sense to add up this value - doing so with the sample data
+would result in a 215% profit margin which (unfortunately) is not correct. We
+can add the `profit_margin_pct` field to our schema like this:
+
+```javascript
+cube(`LineItems`, {
+  ...,
+  measures: {
+    ...,
+    profitMargin: {
+      sql: `profit_margin`,
+      type: `number`,
+      format: 'percentage',
+    },
+  },
+  ...,
+});
+```
+
+Another example of a non-additive measure would be a distinct count of
+`product_id`. If we took the distinct count of products sold over a month, and
+then tried to sum the distinct count of products for each individual day and
+compared them, we would not get the same results. We can add the measure like
+this:
+
+```javascript
+cube(`LineItems`, {
+  ...,
+  measures: {
+    ...,
+    countDistinctProducts: {
+      sql: `product_id`,
+      type: `countDistinct`,
+    },
+  },
+  ...,
+});
+```
+
+### How additivity plays a role in pre-aggregation selection
+
+To recap what we've learnt so far:
+
+- "Additive measures" are measures whose values can be added together.
+- "Multiplied" measures are measures that define `hasMany` relations.
+- "Leaf measures" are measures that do not reference any other measures in their
+  definition.
+- A query is "leaf measure additive" if all of its leaf measures are one of:
+  `count`, `sum`, `min`, `max` or `countDistinctApprox`.
+
+Cube.js selects the best available pre-aggregation based on the incoming queries
+it receives via the API.
```

Comment:
I think it may be understood like Cube.js adjusts selection rules on the fly according to the queries received. Maybe it's better to say that it "...takes in account the query parameters when matching it with a pre-aggregation"?